tests/examples/conda-minimal/dependencies.yaml

```
@@ -11,29 +11,34 @@ channels:
   - rapidsai
   - conda-forge
 dependencies:
-  - output_types: [conda, requirements]
-    common:
-      build:
-        - clang=11.1.0
-    specific:
-      - matrix:
-          cuda: "11.5"
-        build:
-          - cuda-python>=11.5,<11.7.1
-      - matrix:
-          cuda: "11.6"
-        build:
-          - cuda-python>=11.6,<11.7.1
-  - output_types: conda
+  build:
     common:
-      build:
-        - spdlog>=1.8.5,<1.9
+      - output_types: [conda, requirements]
+        packages:
+          - clang=11.1.0
+          - spdlog>=1.8.5,<1.9
+      - output_types: conda
+        packages:
+          - pip:
```

I think using a list of `- pip:` requirements typically also requires the actual `pip` package as a dependency.
```suggestion
          - pip
          - pip:
```